client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -19,9 +22,21 @@ const Launchpad: Step = ( { navigation }: LaunchpadProps ) => {
 	const translate = useTranslate();
 	const almostReadyToLaunchText = translate( 'Almost ready to launch' );
 	const siteSlug = useSiteSlugParam();
-
+	const emailVerified = useEmailVerifiedParam();
 	const site = useSite();
 	const launchpadScreenOption = site?.options?.launchpad_screen;
+	const dispatch = useDispatch();
+
+	useEffect( () => {
+		if ( emailVerified ) {
+			const message = translate( 'Email confirmed!' );
+			dispatch(
+				successNotice( message, {
+					duration: 10000,
+				} )
+			);
+		}
+	}, [ emailVerified, translate, dispatch ] );
```

Comment:
I wonder who is adding the `useEmailVerifiedParam` parameter? Can't that entity dispatch this directly instead?

Reply:
I'm adding the `emailVerified` param when the user is redirected to the launchpad screen from `/home`.

Reply:
Ah then we do need a query param because that's another app. Trying to eradicate as many QPs as possible.